pilot/pkg/model/sidecar.go

```diff
@@ -354,7 +354,10 @@ func convertIstioListenerToWrapper(ps *PushContext, configNamespace string,
 		if _, exists := out.listenerHosts[parts[0]]; !exists {
 			out.listenerHosts[parts[0]] = make([]host.Name, 0)
 		}
-
+		if len(parts) < 2 {
```

Comment:
can you please add a simple test for this?

Reply:
I have added it into `TestCreateSidecarScope`, is this ok